ASSIGMENT/src/pages/aadhaar.astro

```diff
@@ -0,0 +1,64 @@
+---
+import MainLayout from '../layouts/MainLayout.astro';
+import Nav from '../components/Nav.astro';
+---
+
+<head>
+    <link rel="stylesheet" type="text/css" href="/assets/styles/aadhaar.css">
+</head>
+<MainLayout>
+    <nav slot="nav">
+        <Nav />
+    </nav>
+    <section>
+        <label for="documentType" class="dev">Document Aadhaar </label>
+        <form id="aadhaarForm">
+           
+           
+            <div id="documentFields">
+                <div class='formContainer' id='aadhaar'>
+                    <label for="documentNumber_aadhaar">Aadhaar Number</label>
+                    <input type="text" id="documentNumber_aadhaar" name="documentNumber" required>
+                    <label for="holdingPersonName_aadhaar">Name</label>
+                    <input type="text" id="holdingPersonName_aadhaar" name="holdingPersonName" required>
+                    <label for="DOB_aadhaar">Date of Birth</label>
```

Comment:
do not use *aadhaar* as label name; it must be generic like *address*  check on all places and change

Reply:
done